documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,654 @@
+|                                  |                                               |
+|    n == 0                        |    zero?(n)                                   |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    n > 0                         |    positive?(n)                               |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    n < 0                         |    negative?(n)                               |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    i % 2 != 0                    |    odd?(i)                                    |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    i % 2 == 0                    |    even?(i)                                   |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    1 + 2 + 3                     |    1 + 2 + 3                                  |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    1 * 2 * 3                     |    1 * 2 * 3                                  |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    5 - 3                         |    5 - 3                                      |
++----------------------------------+-----------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                         |
+|                                  |                                               |
+|    2.3 / 1.7                     |    2.3 / 1.7                                  |
```

Comment:
An important difference is that `1 / 2` is an error in Dylan, so that would be good to show.  

```
1 / 2 => 0.5         1 / 2 => error
                     1.0 / 2 => 0.5
                     truncate/(1, 2) => 0
                     ceiling/(1, 2) => 1
                     floor/(1, 2) => 0
```